src/services/chris_integration.tsx

```diff
@@ -269,23 +256,30 @@ class ChrisIntegration {
           limit: 25,
           offset: 0,
         });
+
         const newSeries: ISeries = {
           covidnetPluginId: plugin.data.id,
           imageName: '',
           imageId: '',
-          predCovid: 0,
-          predPneumonia: 0,
-          predNormal: 0,
+          columnNames: [],
+          columnValues: [],
```

Comment:
For my personal understanding, I was wondering what are advantages and disadvantages of keeping the prediction results in `columnNames` and `columnValues` arrays compared to keeping them in a dictionary where the key is the name of the column and the value is the number such as { predResults: {[key: string]: number}}

Reply:
A dictionary/map would definitely work just as well (if not better). I went off the array solution, as that was what was mentioned during the meeting, but I am very open to changing it depending on opinion of others (wouldn't take very long for a quick change)

Reply:
@colinywu resolving for now, something to consider later down the road.